src/table-filter-label.js

```diff
@@ -6,13 +6,14 @@ import Popper from '@material-ui/core/Popper'
 import Grow from '@material-ui/core/Grow'
 import Paper from '@material-ui/core/Paper'
 import ClickAwayListener from '@material-ui/core/ClickAwayListener'
-import FilterListIcon from '@material-ui/icons/FilterList'
+import FilterListRoundedIcon from '@material-ui/icons/FilterListRounded'
 import { makeStyles } from '@material-ui/core/styles'
 
 import SelectionFilter from './filters/selection-filter'
 import RangeFilter from './filters/range-filter'
 import QuantitaveFilter from './filters/quantitave-filter'
 import QualitativeFilter from './filters/qualitative-filter'
+import { DateRangeFilter } from './filters/date-range-filter'
```

Comment:
this was for demo and experiment purposes, it's not as refined and also still use the vanilla MUI components

Reply:
so do I just move it back where it was? it won't be testable tho.

Reply:
we can keep it here (and introduce as a new filter addition perhaps)